.gitignore

```diff
@@ -14,7 +14,7 @@ bin
 coverage.txt
 
 # Ignore test reports
-reports/
+test-*.xml
```

Comment:
I guess nothing creates this `reports` directory? I saw an occurrence of this in `scripts/run_script_e2e.sh`, but I guess that script is no longer used, right?

https://github.com/redhat-developer/odo/blob/5a39eda50e0e335cb063b7748dae8302014ca2bc/scripts/run_script_e2e.sh#L88

Reply:
No, it was previously used for test on PSI